lib/karafka/base_responder.rb

```diff
@@ -82,16 +82,18 @@ class << self
       # @param topic_name [Symbol, String] name of topic to which we want to respond
       # @param options [Hash] hash with optional configuration details
       def topic(topic_name, options = {})
+        options[:serializer] ||= Karafka::App.config.serializer
+        options[:registered] = true
```

Comment:
This makes the `options.merge(registered: true)` below redundant.

Reply:
@mensfeld ^ this is my only feedback.

Reply:
True! Thanks!